website/content/docs/upgrading/upgrade-to-1.7.4.mdx

```diff
@@ -0,0 +1,48 @@
+---
+layout: docs
+page_title: Upgrading to Vault 1.7.4 - Guides
```

Comment:
Like the 1.6.6 upgrade page, we should put this on the 1.7.0 upgrade page at least, and maybe 1.7.1/1.7.2/1.7.3 (but I'm more on the fence about those).